src/view/components/common/login/forgot-password-component/forgot-password-component.ts

```diff
@@ -77,6 +81,7 @@ export class ForgotPasswordComponent {
             this.forgotPasswordButton = dataI18n['get-new-password'];
             this.cancelButton = dataI18n.cancel;
             this.forgotPasswordConfirmationMessage = dataI18n['an-email-with-instructions-will-be-sent'];
+            this.emailMandatoryFieldError = (dataI18n['error.form.mandatory']).replace('{0}', this.userIdOrEmailLabel);
```

Comment:
Create a Utility class to replace the {0} would be easier to read :)
Just an idea